src/sentry/api/endpoints/project_templates_index.py

```diff
@@ -41,7 +42,9 @@ class OrganizationProjectTemplatesIndexEndpoint(OrganizationEndpoint):
 
     publish_status = {
         "GET": ApiPublishStatus.PRIVATE,
+        "POST": ApiPublishStatus.PRIVATE,
```

Comment:
Should we change this to experimental for now? I feel like it could be useful to publish these in the future

Reply:
Just curious, what's the difference between them other than semantics? I feel like for the time being, this should likely stay private until we know that we want to eventually make this private. generally easier to roll stuff out than back.

Reply:
The enum comments are like:
https://github.com/getsentry/sentry/blob/d59dea9380eb57def2569285dedefe81e4ea425b/src/sentry/api/api_publish_status.py#L11-L13

Experimental also doesn't publish anything like private